docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+/_site/
```

Comment:
Is this file needed for this commit? I am accepting this for now.

Reply:
The `_site` directory is build and populated by Jekyll's generation process.  With a basic Jekyll site hosted on GH pages, this is handled automatically behind the scenes.

If the Jekyll site becomes more sophisticated, the GH setup may not be able to build it automatically, in which case you need to put `_site` under version control and generate and commit it as it gets updated.  The most common scenario for this to become necessary is using Jekyll plugins that are not on GH pages's safe list.  It is possible to use GH Actions to build/"deploy" Jekyll sites, and there exist actions for this in the Marketplace.  (I haven't looked deeply enough to know what limitations they might have.)

Reply:
> Is this file needed for this commit? I am accepting this for now.

That is a Git ignore for a generated directory of files.  That means that we **don't** want to commit those files.